src/scss/misc.scss

```diff
@@ -391,4 +391,12 @@ app-root > #loading {
             }
         }
     }
+
+    &[role="alert"] {
+        ul {
+            padding-left: 40px;
+            padding-inline-start: 40px;
```

Comment:
I have never seen this property used before. Is it needed?

Reply:
`padding-inline-start` may not be needed, I had copied that from the styling bootstrap applied in the web vault when I suggested it to Vince. Can probably be removed.

Reply:
Removed it and its still working as intended. Updated in latest push.